lib/facility_factory.rb

```diff
@@ -0,0 +1,52 @@
+class FacilityFactory
+  attr_reader :locations
+
+  def initialize
+    @locations = []
+  end
+
+  def facility_creator(dmv)
```

Comment:
I'm a bit confused on what this `dmv` argument is we're passing in here. If it's already the data from the service, then we probably don't want to do the comparison operator on lines 9, 11, and 13 because that will cause a lot of API calls and the data could get unwieldy. Maybe facility _creator just calls all there of your create methods: `create_oregon_facilities` and passes them their corresponding data: `DMV.new.or_dmv_office_locations` and then they just all run and add their locations to @locations.